sensors/gps/common.c

```diff
@@ -52,10 +52,10 @@ int gps_serialSetup(int fd, speed_t baud, struct termios *backup)
 	attr.c_iflag = 0;
 	attr.c_oflag = ONLCR;
 	attr.c_cflag = CS8 | CLOCAL;
-	attr.c_lflag &= ~ECHO;
+	attr.c_lflag &= ~(ECHO | ICANON);
```

Comment:
After checking how the data is being processed it came out that there are problems with reading the data in canonical mode. Using ublox 7n receiver it works fine for some amount of time and then data retrieved from `read()` is broken. By broken I mean that only last 6-10 characters of the last line are read by `read()` function. 

Additionally ublox module also have binary communication protocol which requires the input to not be processed prior to ublox driver. If we ever use it in the future (possible with planned ublox m10 module) it will be necessary. So why not do it now.